cloudinstall/single_install.py

```diff
@@ -150,6 +150,15 @@ def create_container_and_wait(self):
                 self.config.cfg_path,
                 'home/ubuntu/.cloud-install'))
             f.write("/var/cache/lxc var/cache/lxc none bind,create=dir\n")
+            # Detect additional charm plugins and make available to the
+            # container.
+            if self.config.getopt('charm_plugins'):
+                plug_dir = os.path.abspath(self.config.getopt('charm_plugins'))
+                plug_base = os.path.basename(plug_dir)
+                f.write("{d} home/ubuntu/{m} "
+                        "none bind,create=dir\n".format(d=plug_dir,
+                                                        m=plug_base))
+
```

Comment:
what happens here if charm_plugin_dir is the same or a subdir of one of the other directories we've already bind mounted? I can imagine people might use .cloud-install…